package.json

```diff
@@ -32,6 +34,7 @@
                 "file-loader": "6.2.0",
                 "fork-ts-checker-webpack-plugin": "7.2.13",
                 "fs-extra": "10.1.0",
+                "gatecrash": "file:packages/gatecrash-1.3.12.tgz",
```

Comment:
what does this package do?

Reply:
That as well isn't related to the PR. Sorry!